app/internal/wm/GioView.java

```diff
@@ -132,6 +134,64 @@ private void setCursor(int id) {
 		setPointerIcon(pointerIcon);
 	}
 
+    private void setBarColor(String t, int color, int luminance) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+            return;
+        }
+
+        Window window = ((Activity) this.getContext()).getWindow();
+        if (window == null) {
```

Comment:
When is window null? If setBarColor is guaranteed to call while the activity is running, let's drop this check and let a NullPointerException alert us if we violate the guarantee.